theurgy.lic

```diff
@@ -120,10 +321,39 @@ get_from_container('wine')
 fput 'pour wine on altar'
 put_in_container('wine')
 
+if DRSkill.getxp('Theurgy') <= 24
```

Comment:
The reason I suggest moving the checks into the individual commune methods is so that the 'main' method is more about flow - let each individual helper decide how it should achieve its goal.

Not having the elsif would also help if, say, you're still under 31/34 after comm_truff - you could then go and comm_kert.